SignalMessaging/Views/OWSTextField.m

```diff
@@ -32,6 +32,24 @@ - (void)ows_applyTheme
     self.keyboardAppearance = (Theme.isDarkThemeEnabled ? UIKeyboardAppearanceDark : UIKeyboardAppearanceDefault);
 }
 
+- (BOOL)becomeFirstResponder
+{
+    BOOL result = [super becomeFirstResponder];
+
+    [self.ows_delegate textFieldDidBecomeFirstResponder:self];
```

Comment:
Should this be:

if (result) {
  [self.ows_delegate textFieldDidBecomeFirstResponder:self];
}